src/formula.jl

```diff
@@ -213,3 +213,23 @@ function Formula(t::Terms)
     append!(rhs.args,t.terms)
     Formula(lhs,rhs)
 end
+
+"""
+    dropterm(f::Formula, trm::Symbol)
+
+Return a copy of `f` without the term `trm`.
+
+# Examples
+```jl
+julia> dropterm(foo ~ 1 + bar + baz, :bar)
+foo ~ 1 + baz
+```
+"""
+function dropterm(f::Formula, trm::Symbol)
```

Comment:
Terms can be expressions, too (interactions terms). I think `Union{Expr,Symbol}` might do it here.